### PR TITLE
Report back

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
r?

**Comment by ScottDowne** [comment]
Right now the spreadsheet it's publishing to is private, Ali, I shared it with your moz email.

https://docs.google.com/spreadsheets/d/1aQbKaGZRJpnsM3_ZICtLMZk9RdJ9hhJw4vZ95zb_NXo/edit#gid=1276969740

Anyone can publish to it, but the data in it is private :D It's also all client side, and no nonsense API keys.

Took me a while to figure it out because of lack of docs, but it is exactly what i was looking for.

**Comment by alicoding** [comment]
You'll need to rebase

File: call-congress.js [hunk 1]
```diff
@@ -31,6 +31,66 @@ document.addEventListener("DOMContentLoaded", function() {
...
+      congressRepValue = congressRepInput.value;
+    }
+    if (fccMemberInput) {
+      fccMemberValue = fccMemberInput.value;
+    }
+    $.ajax({
+      url: "https://docs.google.com/forms/d/1bB4Tals9lofhSsn4yX03zJ0MbPZQnJrb8JGDSnRs-tg/formResponse",
+      data: {
+        // Senator 1
+        "entry.697672033": senator1Value,
```
Comment (alicoding) [inline call-congress.js]: Any explanation about these numbers?

File: index.html [hunk 2]
```diff
@@ -21,7 +22,7 @@
 
 <body>
   <div class="main-container">
-    <div class="content-container page-1">
+    <div class="content-container page-report-back">
```
Comment (alicoding) [inline index.html]: Why this is changed to `page-report-back`?

Comment (ScottDowne) [inline index.html]: Ha, this is me forgetting to remove my test code.

File: call-congress.js (hunk 1 above)
Comment (ScottDowne) [inline call-congress.js]: These are google form ids

File: index.html [hunk 3]
```diff
@@ -130,6 +140,84 @@
...
+              talked to staff
+            </label>
+            <label>
+              <input name="fcc-member" type="radio" value="unsuccessful">
+              unsuccessful
+            </label>
+          </div>
+        </div>
+        <div class="comments-submit-container">
+          <textarea class="comments-input" rows="10" cols="50" placeholder="Comments"></textarea>
```
Comment (alicoding) [inline index.html]: `Add your comments here...`

Comment (alicoding) [inline index.html]: Also add `padding: 15px`

**Comment by alicoding** [comment]
Fix color on `page-calling`

**Pull request merged**